src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/NftUpload.tsx

```diff
@@ -48,8 +51,10 @@ export default function NftUpload({ form }: { form: FormInstance }) {
   const imageUrl = form.getFieldValue('imageUrl')
 
   const validateImageUrl = () => {
-    if (imageUrl === undefined) {
-      return Promise.reject()
+    if (uploading) {
+      return Promise.reject('File uploading.')
```

Comment:
Show error message when trying to submit new NFT reward while the image is still uploading